TypewiseAlert/IAlert.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace TypewiseAlert
+{
+    public interface IAlert
+    {
+        bool IsAlertPublished(Constants.BreachType breachType);
```

Comment:
kindly change the function name "IsAlertPublished" - what information implementer or caller can derive from it...

Reply:
Updated. please check